src/metrics/chaoss.ts

```diff
@@ -520,6 +520,53 @@ FORMAT JSONCompact`;
   });
 };
 
+export const chaossChangeRequestsAcceptedRatio = async (config:QueryConfig) => {
+  config = getMergedConfig(config);
+  const whereClauses: string[] = ["type = 'PullRequestEvent' AND action = 'closed' "];
+  const repoWhereClause = await getRepoWhereClauseForClickhouse(config);
+  if (repoWhereClause) whereClauses.push(repoWhereClause);
+  whereClauses.push(getTimeRangeWhereClauseForClickhouse(config));
+  const sql = `
+SELECT
+  id,
+  argMax(name, time) AS name,
+  ${getGroupArrayInsertAtClauseForClickhouse(config, { key: 'change_requests_accepted_ratio', value: 'ratio' })},
+  ${getGroupArrayInsertAtClauseForClickhouse(config, { key: 'change_requests_accepted', value: 'acount' })},
+  ${getGroupArrayInsertAtClauseForClickhouse(config, { key: 'change_requests_declined', value: 'dcount' })}
+
+FROM
+(
+  SELECT
+    ${getGroupTimeAndIdClauseForClickhouse(config, 'repo')},
+    COUNT() AS count,
+    countIf(pull_merged = 1) AS acount,
+    countIf(pull_merged = 0) AS dcount,
+    round(acount/count,2) AS ratio
```

Comment:
We don't need to `round` here, `getGroupArrayInsertArrayClauseForClickhouse` will round by precision config for you. And you should use `precision` rather than `2` if you want to `round` here.